packages/types/src/lib/cms.ts

```diff
@@ -100,12 +90,20 @@ export interface User {
 }
 export interface Order {
   id: string;
-  paymentGateway: string;
-  status: 'pending' | 'paid' | 'delivered';
-  customerEmail: string;
-  transactionID: string;
-  orderDateTime: string;
-  orderItems?: OrderItem;
+  items: {
+    id: string;
+    name: string;
+    image: string;
+    color: string;
+    size: string;
+    price: number;
+    quantity: number;
+  }[];
+  transaction_id: string;
+  transaction_time: string;
+  payment_method: string;
+  customer_email: string;
+  status: '1' | '2' | '3';
```

Comment:
Why is order defined again in cms.ts if it is already defined in merch.ts 

> export type Order = z.infer<typeof OrderSchema>;

Reply:
These types are written by Payload when `yarn generate:types` script is called, which happens every time run the Payload CMS server for example.

Wasn't sure if this file should be checked in since it's generated.

@chanbakjsd what's your take on this?